src/dao.ts

```diff
@@ -148,6 +148,10 @@ export class DAO implements IStateful<IDAOState> {
   public stakes(options: IStakeQueryOptions = {}): Observable < IStake[] > {
     throw new Error('not implemented')
   }
+
+  public async ethBalance(): Promise < number > {
```

Comment:
I think we want an `Observable<number>` here, in line with the rest of the lib. 

Web3 has websocket support, and we should use that.

another remark: I'd go for a generic `getBalance(account)`, perhaps on the `Arc` object, because we will also need this value for the current user and for theDAO members...

Reply:
```        
return this.web3.eth.subscribe('newBlockHeaders', (err: Error, result: any) => {
      if (err) {
        console.log(err)
      } else {
        const acctPromise = this.web3.eth.getAccounts().then((accts) => {
          const balance = this.web3.eth.getBalance(accts[0]).then((bal) => {
            console.log('user: ', accts[0])
            console.log('balance: ', bal)
            // res.end('new balance for user: ' + bal)
          })
        })
      
      ```

Reply:
Personally, I don't think that getting ETH balance really belongs to the scope of the client library, and it seems unnecessary to constantly observe that. It seems it would be better using web3 to get balances

Reply:
You are saying many things here, @ben-kaufman  :-)

- "would be better to use web3 to get balances". Yes, agreed.  This is what we are doing here.
- "unnecessary to constantly observe" the balances. why do you think that? The frontend should be updated with new balances when they change, shouldn't it?
- this doe snot belong to the cope of the client library. I disagree: it seems to me a common problem that needs a standard solution, so I think it does belong here.